README.md

```diff
@@ -13,17 +13,40 @@ Check out the `examples` folder for detailed descriptions and code examples for:
 
 # Installation
 
-Phoenics can be installed via pip
+## Environment (Optional)
```

Comment:
Recommended not optional

Reply:
I prefer to keep `optional` in the paragraph title. We already say in the description that it is recommended.